### PR TITLE
Stuff

### PR DESCRIPTION
**Pull request by prosif**
Add new squish functionality to support complex shapes, text color and styling, node effects, probably some other stuff too

### CODE REVIEW

**Comment by prosif** [comment]
Need to rebase this on your changes @YazeedLoonat, but I'm putting it up for review because its not going to change much with the rebase

**Comment by prosif** [comment]
Rebased on master

File: package.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "squishjs",
-  "version": "0.4.9",
+  "version": "0.5",
   "description": "squish & unsquish stuff",
   "scripts": {
-    "test": "node testRunner.js && node ./test/test.js"
+    "test": "node test.js"
```
Comment (YazeedLoonat) [inline package.json]: I think the change here is a mistake to what was in master
We should use the testRunner here and the ./test/test.js file should probably be changed to use the testRunner instead

**Review by YazeedLoonat — CHANGES_REQUESTED** [review]
Hey I think the rebase caused a few problems
1) with tests, the changes to what npm test does are a rollback of the changes that were in master
2) usage of the Colors.js file seem to be using the older way this file functioned

Examples being importing `Colors` and then trying to use `colors.Black` which doesn't exist anymore and should instead be `Colors.Colors.Black` 

3) this is also failing CI because of those changes. Can you address or I can take a peak in and start to make the updates

**Comment by prosif** [comment]
Updated to fix tests as well as a bug with the ShapeUtils array dimension

**Comment by prosif** [comment]
@YazeedLoonat is this good to merge? I have more changes based on this branch and I don't want to create a chain of PRs

**Pull request merged**